website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -405,10 +405,24 @@ The examples described in this section demonstrate how to export a service named
 ## End a peering connection
 
 To end a peering connection, delete both the `PeeringAcceptor` and `PeeringDialer` resources.
+ 
+1. Delete the `PeeringDialer` resource from the second cluster.
 
-To confirm that you deleted your peering connection, in `cluster-01`, query the `/health` HTTP endpoint. The peered services should no longer appear.
+  ```shell-session
+  $ kubectl --context $CLUSTER2_CONTEXT delete --filename dialer.yaml 
+  ```
+
+1. Delete the `PeeringAcceptor` resource from the first cluster.
+
+    ```shell-session
+    $ kubectl --context $CLUSTER1_CONTEXT delete --filename acceptor.yaml
+    ````
+
+1. Confirm that you deleted your peering connection, in `cluster-01`, by querying the the `/health` HTTP endpoint. The peered services should no longer appear.
 
 ```shell-session
+$ kubectl exec -it consul-server-0 -- /bin/sh
+$ export CONSUL_HTTP_TOKEN=<INSERT BOOTSTRAP ACL TOKEN> # export ACL token if ACLs are enabled
 $ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
 ```
```

Comment:
````suggestion      

     Enter the first cluster.
     
     ```shell-session
     $ kubectl exec -it consul-server-0 -- /bin/sh
     ```
     
     If you have ACLs enabled, export them as environment variables.

     ```shell-session
     $ export CONSUL_HTTP_TOKEN=<INSERT BOOTSTRAP ACL TOKEN>
     ```

     Query the `/health` endpoint to confirm there are no peered services.     

     ```shell-session
     $ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
     ```
````

suggestion to divide the commands so the copy button works. also, moved the codeblock so it's indented properly